src/main/java/net/pms/util/ProcessUtil.java

```diff
@@ -266,14 +293,14 @@ public static void reboot(ArrayList<String> cmd, Map<String,String> env, String
 			startdir = System.getProperty("user.dir");
 		}
 
-		System.out.println("starting: " + StringUtils.join(cmd, " "));
+		System.out.println("Starting: " + StringUtils.join(cmd, " "));
 
 		final ProcessBuilder pb = new ProcessBuilder(cmd);
 		if (env != null) {
 			pb.environment().putAll(env);
 		}
 		pb.directory(new File(startdir));
-		System.out.println("in directory: " + pb.directory());
+		System.out.println("In folder: " + pb.directory());
```

Comment:
Make me remember to don't ask you to capitalize my money :rofl: 

Thanks to improve/enhance my english ;-)